packages/components/pages/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -3133,6 +3135,62 @@ describe('Checkout', () => {
             });
         });
 
+        describe('submitOrder (split notes enabled)::', () => {
+            const basketId = 'myBasketId-v1';
+            let wrapper;
+
+            beforeEach(() => {
+                wrapper = shallowMount(VueCheckout, {
+                    store: createStore({
+                        ...defaultCheckoutState,
+                        notes: { courier: { note: 'This is a courier note' }, kitchen: { note: 'This is a kitchen note' } },
+                        notesConfiguration: { isSplitNotesEnabled: true },
+                        basket: {
+                            id: basketId
+                        }
+                    }),
+                    i18n,
+                    localVue,
+                    propsData,
+                    mocks: {
+                        $logger,
+                        $cookies
+                    }
+                });
+            });
+
+            afterEach(() => {
+                jest.clearAllMocks();
+            });
+
+            it('should call `placeOrder`', async () => {
+                // Arrange
+                const placeOrderSpy = jest.spyOn(wrapper.vm, 'placeOrder');
+                jest.spyOn(wrapper.vm, 'getReferralState').mockImplementation(() => 'MockReferralState');
```

Comment:
Would it be better to mock the cookie value instead, so that `getReferralState` doesn't need mocking?

Reply:
I actually didn't write this test, I just moved it under the describe block 'submitOrder (split notes enabled)::'